plutus-benchmark/plutus-benchmark.cabal

```diff
@@ -550,13 +550,25 @@ test-suite plutus-benchmark-marlowe-tests
 
 -- TODO: Add benchmarks for the executable semantics when we have a UPLC version
 
+library agda-internal
+  import:          lang, ghc-version-support
```

Comment:
You included `ghc-version-support`, which marks things as unbuildable for some GHCs. That's not true here, and it's breaking things since the agda benchmarks are buildable on all GHCs.

Reply:
I have to say that the nix error message isn't very illuminating.

Reply:
Well, it's a cabal error message, and it tells you that this component isn't buildable. I looked for reasons why it might not be buildable, that's how I found it :shrug:

Reply:
> Well, it's a cabal error message,

Oh. That's not what I was getting earlier, or if it was it was buried inside [2798 lines of nix crap](https://ci.iog.io/eval/39146#tabs-errors).

Reply:
OK, I think I understand what's happening now and I've managed to sort it all out.  Thanks for the pointers.